doc/devdocs/modules/fancyzones.md

```diff
@@ -1,85 +1,25 @@
-## FancyZones Lib
+# FancyZones UI tests
 
-#### [`FancyZones.cpp`](/src/modules/fancyzones/lib/FancyZones.cpp)
-TODO
+UI tests are implemented using [Windows Application Driver](https://github.com/microsoft/WinAppDriver).
 
-#### [`Settings.cpp`](/src/modules/fancyzones/lib/Settings.cpp)
-TODO
+## Before running tests
 
-#### [`trace.cpp`](/src/modules/fancyzones/lib/trace.cpp)
-TODO
+  - Download and run Windows Application Driver installer from https://github.com/Microsoft/WinAppDriver/releases
```

Comment:
Which version?

Reply:
I updated the doc and specified the version.